### PR TITLE
Added Husky Workflow

### PR DESCRIPTION
**Pull request by VanshikaSabharwal**
Note- This husky workflow will only run when you commit changes.

You can now add merge this to the main branch @ashutosh-rath02 .

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
Hey @VanshikaSabharwal please remove changes from README.md and resolve the conflicts please. I think you haven't pulled the recent updates.

File: components/AuthButton.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default function AuthButton({ user, height, width }: AuthButtonProps) {
   ) : (
     <Button
       variant="outline"
-      className="inline-flex h-11 animate-shimmer items-center justify-center rounded-md border-2 border-slate-800 bg-[linear-gradient(110deg,#000103,45%,#1e2631,55%,#000103)] bg-[length:200%_100%] px-12 font-medium  text-white text-lg transition-colors focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50"
+      className="inline-flex h-11 animate-shimmer items-center justify-center rounded-md border-2 border-slate-800 bg-[linear-gradient(110deg,#000103,45%,#1e2631,55%,#000103)] hover:text-white  bg-[length:200%_100%] px-12 font-medium  text-white text-lg transition-colors focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50 "
```
Comment (ashutosh-rath02) [inline components/AuthButton.tsx:45]: remove the changes here. Not part of the PR

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

File: components/AuthButton.tsx (hunk 1 above)
Comment (VanshikaSabharwal) [reply components/AuthButton.tsx]: Yes, I did it

**Comment by ashutosh-rath02** [comment]
@VanshikaSabharwal please resolve the merge conflicts

**Comment by VanshikaSabharwal** [comment]
> Hey @VanshikaSabharwal please remove changes from README.md and resolve the conflicts please. I think you haven't pulled the recent updates. Can you please share a screenshot of the running workflow from your terminal.


Screenshot 2024-05-11 141209

**Comment by VanshikaSabharwal** [comment]
> @VanshikaSabharwal please resolve the merge conflicts

Yes i have resolved the conflict

**Comment by ashutosh-rath02** [comment]
@all-contributors please add @VanshikaSabharwal for code

LGTM🧨
